### PR TITLE
Static Webapp CRUD

### PR DESCRIPTION
**Pull request by o-reo**
Added CRUD operation for azure static webapps
## How to
To test the commands you need to add the following keys in your config
```yaml
# WebApp azure location
webapp_location: ""
# WebApp Github Repository
webapp_repository: ""
# WebApp Github Repository branch
webapp_repository_branch: "main"
# WebApp Github Repository Token (will be moved in secrets)
webapp_repository_token: ""
```
## Commands
```bash
babylon azure staticwebapp get-all
babylon azure staticwebapp create babylon_test -e -f webapp_details.json
babylon azure staticwebapp update babylon_test -e -f webapp_details.json
babylon azure staticwebapp get babylon_test
babylon azure staticwebapp delete babylon_test
babylon azure staticwebapp custom-domain create babylon_test my.domain.net
babylon azure staticwebapp custom-domain get-all babylon_test
babylon azure staticwebapp custom-domain get babylon_test my.domain.net
babylon azure staticwebapp custom-domain delete babylon_test my.domain.net
babylon azure staticwebapp app-settings update babylon_test -e -f webapp_settings.json
babylon azure staticwebapp app-settings get babylon_test
```
Note that static web apps need time to effectively create

I also removed the `powerbi_api_scope` as I found it's better to put in hard text than in config, it is not supposed to change

### CODE REVIEW

File: Babylon/groups/powerbi/__init__.py [hunk 1]
```diff
@@ -8,16 +8,14 @@
...
 
 @group()
 @pass_context
-@require_platform_key("powerbi_api_scope")
-def powerbi(ctx: Context, powerbi_api_scope: str):
+def powerbi(ctx: Context):
     """Group handling communication with PowerBI API"""
     try:
-        token = DefaultAzureCredential().get_token(powerbi_api_scope)
+        token = DefaultAzureCredential().get_token("https://analysis.windows.net/powerbi/api/.default")
```
Comment (neomatamune) [inline Babylon/groups/powerbi/__init__.py:18]: Why  replacing configurable value with fixed one ?

**Review by neomatamune — CHANGES_REQUESTED** [review]

File: Babylon/groups/powerbi/__init__.py (hunk 1 above)
Comment (o-reo) [reply Babylon/groups/powerbi/__init__.py:18]: This is not supposed to be a user configurable variable. It's a windows api url.

Comment (o-reo) [reply Babylon/groups/powerbi/__init__.py:18]: But we can put it in a project config file somewhere, something like routes.py

Comment (o-reo) [reply Babylon/groups/powerbi/__init__.py]: I moved this in another branch to have a dedicated rework

**Review by neomatamune — APPROVED** [review]

**Pull request merged**